nodejs/week1/Node-week1-Homework/warmup/warmup.js

```diff
@@ -0,0 +1,33 @@
+// Circle Calculator.
+
+class Circle {
+	constructor(radius) {
+		this.radius = radius;
+	}
+	getDiameter() {
+		const diameter = this.radius * 2;
+		console.log(`The diameter of the circle is ${diameter}`);
+	}
+	getCircumference() {
+		const circumference = Math.PI * this.radius * 2;
+		console.log(
+			`The circumference of the circle is ${circumference.toFixed(3)}`
+		);
+	}
+
+	getArea() {
+		const area = Math.PI * (this.radius * this.radius);
```

Comment:
It is a good idea to change `this.radius * this.radius `by `Math.pow(this.radius, 2)` (to not repeat yourself)
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/pow

Reply:
Okay, thanks for the link